app/src/views/animations/status-animations.js

```diff
@@ -92,9 +121,9 @@ export default class StatusAnimations {
   }
 
   static clearView(view) {
-    //TODO: This code relies on there being certain hard coded stripIds
-    //TODO: All of this should be using a config - that would then reflect
-    //TODO: how the light array gets setup in the store
+    // TODO: This code relies on there being certain hard coded stripIds
+    // TODO: All of this should be using a config - that would then reflect
+    // TODO: how the light array gets setup in the store
```

Comment:
I believe we have a config variable for this. So this TODO needs to be resolved.